MovieJournalMVC/View/MovieReviewCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  MovieReviewCell.swift
+//  MovieJournalMVC
+//
+//  Created by Lauren Small on 3/12/19.
+//  Copyright © 2019 Lauren Small. All rights reserved.
+//
+
+import UIKit
+
+class MovieReviewCell: UITableViewCell {
+
+    
+    @IBOutlet weak var movieTitleLabel: UILabel!
+    var title: String?
+    var review: String?
+    
+    
+    
+    func getCellText(entry: String) {
```

Comment:
The naming here isn't clear. Something like `setup(using review: MovieReview)` could be used.

Reply:
I renamed the function to your suggestion, changing the parameter from accepting a String value to accept a MovieReview instead.

func setup(using review: MovieReview) {
        movieTitleLabel.text = review.title
    }